tests/large/test_ide.py

```diff
@@ -326,6 +326,21 @@ def setUp(self):
         self.name = 'DataGrip'
 
 
+class RiderIDETests(IdeaIDETests):
+    """Rider from the IDE collection."""
+
+    TIMEOUT_INSTALL_PROGRESS = 120
+    TIMEOUT_START = 60
+    TIMEOUT_STOP = 60
+
+    def setUp(self):
+        super().setUp()
+        self.installed_path = os.path.join(self.install_base_path, "ide", "rider")
+        self.desktop_filename = 'jetbrains-rider.desktop'
+        self.command_args = '{} ide rider'.format(UMAKE)
+        self.name = 'Rider'
```

Comment:
Isn't it done here [](https://github.com/ubuntu/ubuntu-make/blob/5910bdad1742f1b356927e675c6453f743e3cc05/tests/large/test_ide.py#L163)